tests/unit/s2n_session_ticket_test.c

```diff
@@ -474,44 +492,44 @@ int main(int argc, char **argv)
         EXPECT_NOT_NULL(server_config = s2n_config_new());
         EXPECT_SUCCESS(s2n_config_set_session_tickets_onoff(server_config, 1));
 
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, tick_key_name1, S2N_TICKET_KEY_NAME_LEN, tick_key1, sizeof(tick_key1), 0));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, S2N_TICKET_KEY_NAME_LEN, ticket_key1, sizeof(ticket_key1), 0));
```

Comment:
why sizeof(ticket_key1) and not strlen(ticket_key1)?

same for all subsequent ones

Reply:
Done.